cfme/tests/cloud_infra_common/test_relationships.py

```diff
@@ -136,3 +143,19 @@ def test_cloud_provider_relationships(appliance, provider, setup_provider, relat
     provider_view.entities.summary("Relationships").click_at(relationship)
     relationship_view = appliance.browser.create_view(view, additional_context={'object': obj})
     assert relationship_view.is_displayed
+
+
+@pytest.mark.provider([AzureProvider, EC2Provider, OpenStackProvider, GCEProvider])
+def test_cloud_provider_refresh_relationship(provider, setup_provider):
+    provider_view = navigate_to(provider, 'Details')
+    provider_view.toolbar.configuration.item_select("Refresh Relationships and Power States",
+                                                    handle_alert=True)
+    wait_for_cloud_relationship_refresh(provider)
+
+
+@pytest.mark.provider([VMwareProvider, RHEVMProvider, SCVMMProvider])
```

Comment:
**Optional**: Can be done as one test case for cloud infra with adding 
@pytest.mark.uncollectif(lambda provider: not provider.one_of(EC2Provider)) if some of the providers should be skipped

Reply:
Sure, will make it as one test case.